tasks/push-snapshot/push-snapshot.yaml

```diff
@@ -65,14 +81,17 @@ spec:
             exit 1
         fi
 
-        floatingTag=$(jq -r '.images.floatingTag // ""' $DATA_FILE)
+        floatingTagsCount=$(jq -rc '.images.floatingTags | length' $DATA_FILE)
         timestampFormat=$(jq -r '.images.timestampFormat // "%s"' $DATA_FILE)
         timestamp="$(date "+$timestampFormat")"
-        if [ -n $floatingTag ]; then
-            echo -n "${floatingTag}-${timestamp} ${floatingTag}" > $(results.commonTags.path)
-        else
-            echo -n "" > $(results.commonTags.path)
+        commonTags=""
+        if [ $floatingTagsCount -gt 0 ]; then
+            for floatingTag in $(jq -rc '.images.floatingTags[]' $DATA_FILE) ; do
```

Comment:
`-c` is redundant, the result is a list of strings (one per line), so nothing to compress here.